speech-to-text/v1-generated.ts

```diff
@@ -2807,16 +2886,23 @@ class SpeechToTextV1 extends BaseService {
    * or a list of words that are relevant to the contents of the audio files. Both of the custom models must be based on
    * the same version of the same base model for training to succeed.
    *
-   * Training can fail to start for the following reasons:
+   * **See also:**
+   * * [Train the custom acoustic
+   * model](https://cloud.ibm.com/docs/services/speech-to-text?topic=speech-to-text-acoustic#trainModel-acoustic)
+   * * [Using custom acoustic and custom language models
+   * together](https://cloud.ibm.com/docs/services/speech-to-text?topic=speech-to-text-useBoth#useBoth)
+   *
+   * ### Training failures
+   *
+   *  Training can fail to start for the following reasons:
    * * The service is currently handling another request for the custom model, such as another training request or a
    * request to add audio resources to the model.
-   * * The custom model contains less than 10 minutes or more than 100 hours of audio data.
-   * * One or more of the custom model's audio resources is invalid.
+   * * The custom model contains less than 10 minutes or more than 200 hours of audio data.
    * * You passed an incompatible custom language model with the `custom_language_model_id` query parameter. Both custom
    * models must be based on the same version of the same base model.
-   *
-   * **See also:** [Train the custom acoustic
-   * model](https://cloud.ibm.com/docs/services/speech-to-text/acoustic-create.html#trainModel-acoustic).
+   * * The custom model contains one or more invalid audio resources. You can correct the invalid audio resources or set
+   * the `strict` parameter to `false` to exclude the invalid resources from the training. The model must contain at
```

Comment:
I think we need to take out mentioning of `strict` parameter?

Reply:
I think you're right. I'm about to meet with Taj about any extra changes like that. If possible, I want to try and make some changes in `v1.ts` rather than `v1-generated.ts` to avoid hand-written changes in the future